packages/create-react-app-demo/src/docs-chat/ai.tsx

```diff
@@ -19,18 +18,27 @@ export interface ChatMessage {
 export const conversationAtom = atom<ChatMessage[]>([]);
 export const modelCallInProgress = atom<boolean>(false);
 
-// For now, we load the ai.jsx docs from a local markdown file. Once we have a HTML->Markdown converter,
-// we can load the docs from the site directly.
-const docResponse = await fetch(markdownPath);
-const docText = await docResponse.text();
-const docs = [
-  {
-    pageContent: [docText],
-    name: 'Guide for AI Newcomers',
-  },
-];
-const corpus = new LocalCorpus(staticLoader(docs), defaultChunker);
-await corpus.startLoading();
+/*
+ * This is a very simple example of how to build a document corpus.
+ * We pull the page content from a URL, convert it to Markdown, and then index it.
```

Comment:
This is fine for the example, but if we're converting to md, should we just request the raw file from GH directly? https://raw.githubusercontent.com/fixie-ai/ai-jsx/main/packages/docs/docs/guides/brand-new.md